copyright.txt

```diff
@@ -1,4 +1,5 @@
 // Copyright 2015-2017 Benjamin Fry <benjaminfry@me.com>
+// Copyright 2017 Google LLC.
```

Comment:
this file just holds the text I use to copy and paste into files I create... It shouldn't need the Google notice.